### PR TITLE
Attempting to fix training cycle

### PR DESCRIPTION
**Pull request by gennarinoos**

### CODE REVIEW

**Comment by martinnormark** [comment]
I managed to get the training script running after quite a number of changes. It turned out there was a bunch of mismatches in how parameters were given to loss functions etc. Main fixes are here: https://github.com/martinnormark/neural-mesh-simplification/commit/3d8d35d26a7e9510d7ceb2cb383609af093bb582
